packages/integration_test/android/build.gradle

```diff
@@ -26,7 +26,8 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 31
+    namespace = 'dev.flutter.integration_test'
```

Comment:
I know almost nothing about Groovy, so maybe this is an irrelevant difference, but I've been doing using the syntax `namespace 'dev.flutter.integration_test'`, which is what https://developer.android.com/build/configure-app-module#groovy shows for Groovy.

I see looking around more that https://developer.android.com/build/publish-library/prep-lib-release does the `=`, but maybe there's a difference in how unsupported/unknown things are handled with one syntax vs. the other?

Reply:
yeah given that groovy is a dynamic language I wonder if the `=` is more strict.  That seems like a reasonable guess.

Reply:
My understanding of groovy is that `foo 'bar'` is equivalent to `foo('bar')` (i.e. a method call), while `foo = 'bar'` is what you would expect it to be. 

But that they end up equivalent in some places because that `foo` is just a setter method for a `foo` property.

See https://stackoverflow.com/questions/22759542/confused-with-the-use-of-assignment-and-brackets (the question is somewhat different but the top answer covers the topic).